WalletWasabi.Fluent/Controls/DualCurrencyEntryBox.axaml.cs

```diff
@@ -372,15 +372,20 @@ private void ReorganizeVisuals()
 
 			if (IsConversionReversed)
 			{
-				_leftEntryBox.HorizontalAlignment = Avalonia.Layout.HorizontalAlignment.Right;
-				_rightEntryBox.HorizontalAlignment = Avalonia.Layout.HorizontalAlignment.Stretch;
+				_leftEntryBox.TextAlignment = Avalonia.Media.TextAlignment.Right;
```

Comment:
Add a psuedo class: :isbtc :isfiat 

and then set those in a style like : `Selector="DualCurrencyEntryBox:isFiat" Setter="Property ="HorizontalContentAligntment="x"

Reply:
`PseudoClasses.Set(":expanded", value);`